boards/waspmote-pro/include/periph_conf.h

```diff
@@ -104,9 +104,27 @@ extern "C" {
 /** @} */
 
 /**
- * @brief    I2C configuration
+ * @name    I2C configuration
+ * @{
  */
 #define I2C_NUMOF           1
+/** @} */
+
+/**
+ * @name     ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           8
+/* ADC Channels */
+#define ADC_PIN_0           0
+#define ADC_PIN_1           1
+#define ADC_PIN_2           2
+#define ADC_PIN_3           3
+#define ADC_PIN_4           4
+#define ADC_PIN_5           5
+#define ADC_PIN_6           6
+#define ADC_PIN_7           7
```

Comment:
@PeterKietzmann I think these names are from according to the waspmote manual they should be

```
ANALOG1 => 14 
ANALOG2 => 15
ANALOG3 => 16 
ANALOG4 => 17 
ANALOG5 => 18 
ANALOG6 => 19 
ANALOG7 => 20
```

right? Though without the values, its just copy-paste from page 46.

Reply:
@PeterKietzmann looking at page 45 of the waspmote manual there is a `ANALOG` pin top-left on the larger connector, maybe that's the missing ANALOG0?

Reply:
Actually I don't even know why these 'pins' are defined here. `ADC_DEV(x)` returns `x` and should be used AFAIK. The ANALOG pin on the top left of the waspmote does not work. I'm against introducing the waspmote semantic to RIOTs periph_conf.h

Reply:
but then something like `val = analogRead(ANALOG1);` as in the manual is not working.

Reply:
when using an Arduino sketch with RIOT

Reply:
Now that you addressed several arduino mappings in this PR, what is this needed for?